sockeye/image_captioning/inference.py

```diff
@@ -128,8 +128,8 @@ def translate(self, trans_inputs: List[TranslatorInput]) -> List[TranslatorOutpu
                 if self.source_root is not None:
                     path = os.path.join(self.source_root, path)
                 paths.append(path)
-            batch_translations = self._translate_nd(*self._get_inference_input(paths))
             # truncate to remove filler translations
+            batch_translations = self._translate_nd(*self._get_inference_input(paths))
```

Comment:
seems like an unnecessary change

Reply:
the reordering with the comment? I agree.